helm_deploy/hmpps-approved-premises-api/values.yaml

```diff
@@ -50,6 +50,8 @@ generic-service:
       SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_HMPPS-TIER_CLIENT-SECRET: "SYSTEM_CLIENT_TIER_SECRET"
       SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_PRISONS-API_CLIENT-ID: "SYSTEM_CLIENT_NOMIS_ID"
       SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_PRISONS-API_CLIENT-SECRET: "SYSTEM_CLIENT_NOMIS_SECRET"
+      SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_MANAGE-ADJUDICATIONS-API_CLIENT-ID: "SYSTEM_CLIENT_NOMIS_ID"
```

Comment:
See [Jira comment](https://dsdmoj.atlassian.net/browse/NN-5363?focusedCommentId=317904) for roles which have been updated

Reply:
From reading back on the notes from the problems on [the first release of this](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/882) it seems like [the core problem was our requests not being authorised](https://mojdt.slack.com/archives/C012FNNQ4A3/p1692093856763789?thread_ts=1692009909.588849&cid=C012FNNQ4A3).

Was that because we didn't have this variable set last time so when [we set up the previous client](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/882/files#diff-61e5abfecfb05b57d0af67e97f31c84fde2d4f9033881ac609694b0d77c78178R190) it instantiated without the correct keys. Basically, are we confident that this was the problem?

Reply:
Yes - I missed that bit of config from my first PR